images/linux/scripts/installers/kotlin.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash -e
+################################################################################
+##  File:  kotlin.sh
+##  Desc:  Installs Kotlin
+################################################################################
+
+source $HELPER_SCRIPTS/install.sh
+
+URL=$(curl -s https://api.github.com/repos/JetBrains/kotlin/releases/latest | jq -r '.assets[].browser_download_url | select(contains("kotlin-compiler"))')
+download_with_retries $URL "/tmp"
+
+unzip -qq /tmp/kotlin-compiler*.zip -d /tmp
+FILES="/tmp/kotlinc/bin/*"
+
+for f in $FILES
+do
+    if [[ "$f" != *".bat"* ]]; then
```

Comment:
Looks like we can simply delete all the bat files instead as it was done in the homebrew formula
https://github.com/Homebrew/homebrew-core/blob/HEAD/Formula/kotlin.rb#L21